network/definition/data/components/vcrear/vcrear-message.yaml

```
@@ -76,3 +76,17 @@ messages:
       shockpotdispRL:
       shockpotVoltRR:
       shockpotdispRR:
+
+  wheelSpeed:
+    description: Rear wheel speeds
+    id: 0x460
+    cycleTimeMs: 50
+    signals:
+      wheelVelocityL:
+        template: wheelLinearVelocity
+      wheelVelocityR:
+        template: wheelLinearVelocity
```

Based on your templates, these aren't actually velocities, they're magnitudes.
I'd go with
```suggestion
      wheelSpeedLinearL:
        template: wheelLinearVelocity
      wheelSpeedLinearR:
        template: wheelLinearVelocity
```